fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml

```diff
@@ -0,0 +1,15 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: longhorn
+  namespace: fleet-default
+spec:
+  repo: https://github.com/suse-edge/misc.git
+  branch: longhorn-fleet-default
```

Comment:
Is this intentionally not `main`?

Reply:
Good catch, thanks! I forgot to put it back to main :D